pkg/nuctl/command/common/importreport.go

```diff
@@ -56,6 +57,13 @@ func (pr *ProjectReports) SprintfError() string {
 	return report
 }
 
+func (pr *ProjectReports) PrintAsTable(t table.Writer, onlyFailed bool) {
+	t.AppendHeader(table.Row{"type", "name", "status", "fail description", "auto-fixable"})
+	for _, report := range pr.Reports {
+		report.PrintAsTable(t, onlyFailed)
+	}
+}
```

Comment:
Perhaps you can reuse `renderer.RenderTable`

Reply:
@TomerShor we do really need between-project separator, so I'd leave as is. wdyt?

Reply:
ok